source/unified-test-format/tests/invalid/expectedTracingSpans-clientType.yml

```diff
@@ -0,0 +1,14 @@
+description: "expectedTracingSpans-clientType"
+
+schemaVersion: "1.26"
+
+createEntities:
+  - client:
+      id: &client0 "client0"
+
+tests:
+  - description: "client type must be string"
+    operations: []
+    expectTracingMessages:
+      client: 0
+      spans: [{}]
```

Comment:
Same as above. This should have a valid span object to ensure `client` is actually what causes a validation error.